### PR TITLE
Add experimental 3.11 tests

### PR DESCRIPTION
**Pull request by Dreamsorcerer**

### CODE REVIEW

**Comment by Dreamsorcerer** [comment]
Well, that's weird. pytest just fails with no output...

**Comment by cclauss** [comment]
Try to re-run on beta 4 instead of beta 3.

**Comment by cclauss** [comment]
Will `Python 3.11-beta.4` pass if the `cython==0.29.30` upgrade is made in the file ``requirements/ci.txt`?

**Comment by Dreamsorcerer** [comment]
I'll try it out later, when I've got time. But, feel free to branch off and get it working before me.

**Comment by cclauss** [comment]
The GitHub Actions on my PRs (like #306, 307, and 308) do not run until approved by a repo maintainer.

**Comment by Dreamsorcerer** [comment]
Approved them.

**Comment by cclauss** [comment]
Can you please re-approve #308

I believe that #307 is ready to be reviewed and merged.

**Comment by cclauss** [comment]
Upgrading all dependencies in requirements/ci.txt the GitHub Action tests to pass on Python 3.11.

**Comment by cclauss** [comment]
The failing test is being run on Collecting pytest==6.2.5, not the current version.  https://pypi.org/project/pytest

**Comment by Dreamsorcerer** [comment]
OK, dependabot needs to be fixed for this project. It's only updating things in the github workflow, not Python requirements files..

**Comment by webknjaz** [comment]
> OK, dependabot needs to be fixed for this project. It's only updating things in the github workflow, not Python requirements files..

Yeah, I noticed that but couldn't figure out why that's happening, looking at the config. Does it need to be pointed at a different folder?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -74,13 +74,20 @@ jobs:
...
+        experimental: [false]
         exclude:
           - os: macos
             no-extensions: 'Y'
           - os: windows
             no-extensions: 'Y'
+        include:
+          - os: ubuntu
+            pyver: "3.11.0-alpha - 3.11.0"
+            experimental: true
```
Comment (webknjaz) [inline .github/workflows/ci.yml:86]: FWIW, if it passes, maybe we don't need "experimental" anymore? The first release candidate is just a week away, after all...

Comment (Dreamsorcerer) [reply .github/workflows/ci.yml:86]: I'm inclined to just leave an experimental version in, so it can just be bumped to 3.12 a little later.

**Comment by Dreamsorcerer** [comment]
> Yeah, I noticed that but couldn't figure out why that's happening, looking at the config. Does it need to be pointed at a different folder?

It appears to be the same in aiohttp, so no idea why it's not activating here.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (webknjaz) [reply .github/workflows/ci.yml:86]: Too late https://github.com/aio-libs/frozenlist/commit/15b00857f004dc44ab9610695026188868bddb0e

**Pull request merged**